modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -82,36 +83,43 @@ DEFAULT pxeboot
 TIMEOUT 20
 PROMPT 0
 LABEL pxeboot
-    KERNEL http://<HTTP_server>/rhcos-<version>-installer-live-kernel-<architecture> <1>
-    APPEND initrd=http://<HTTP_server>/rhcos-<version>-installer-live-initramfs.<architecture>.img console=ttyS0 console=tty0 coreos.inst.install_dev=/dev/sda coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign coreos.live.rootfs_url=http://<HTTP_server>/rhcos-<version>-installer-live-rootfs.<architecture>.img <2>
+    KERNEL http://<HTTP_server>/rhcos-<version>-live-kernel-<architecture> <1>
+    APPEND initrd=http://<HTTP_server>/rhcos-<version>-live-initramfs.<architecture>.img coreos.live.rootfs_url=http://<HTTP_server>/rhcos-<version>-live-rootfs.<architecture>.img coreos.inst.install_dev=/dev/sda coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <2> <3>
 ----
 <1> Specify the location of the live `kernel` file that you uploaded to your HTTP
 server.
-<2> Specify locations of the {op-system} files that you uploaded to your HTTP
-server. The `initrd` parameter value is the location of the live `initramfs`
-file, the `coreos.inst.ignition_url` parameter value is the location of the
-bootstrap Ignition config file, and the `coreos.live.rootfs_url` parameter value is
-the location of the live `rootfs` file.
+The URL must be http, tftp, or ftp; https and nfs are not supported.
+<2> If you use multiple NICs, specify a single interface in the `ip` option.
+For example, to use DHCP on a NIC that is named `eno1`, set `ip=eno1:dhcp`.
+<3> Specify locations of the {op-system} files that you uploaded to your
+HTTP server. The `initrd` parameter value is the location of the `initramfs` file,
+the `coreos.live.rootfs_url` parameter value is the location of the
+rootfs file, and the `coreos.inst.ignition_url` parameter value is the
+location of the bootstrap Ignition config file.
+You can also add more kernel arguments to the APPEND line to configure networking
+or other boot options.
 
 ** For iPXE:
 +
 ----
-kernel http://<HTTP_server>/rhcos-<version>-installer-kernel-<architecture> console=ttyS0 console=tty0 coreos.inst.install_dev=/dev/sda coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign coreos.live.rootfs_url=http://<HTTP_server>/rhcos-<version>-installer-live-rootfs.<architecture>.img <1>
-initrd=http://<HTTP_server>/rhcos-<version>-installer-live-initramfs.<architecture>.img <2>
+kernel  http://<HTTP_server>/rhcos-<version>-live-kernel-<architecture> coreos.live.rootfs_url=http://<HTTP_server>/rhcos-<version>-live-rootfs.<architecture>.img coreos.inst.install_dev=/dev/sda coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <1> <2>
+initrd=http://<HTTP_server>/rhcos-<version>-live-initramfs.<architecture>.img
```

Comment:
This should either be moved into the previous line, or the `=` replaced with a space.

Reply:
I replaced with = with a space.